ting_subsearch_suggestions.module

```diff
@@ -179,44 +184,68 @@ function ting_subsearch_suggestions_cron() {
     }
 
     $url = variable_get('ting_subsearch_kpi_webservice_endpoint', NULL);
-    $params['start_period'] = variable_get('ting_subsearch_kpi_start_period', NULL);
-    $params['end_period'] = variable_get('ting_subsearch_kpi_end_period', NULL);
-    $params['format'] = variable_get('subsearch_kpi_format', NULL);
-    $params['language'] = variable_get('subsearch_kpi_language', NULL);
 
-    $account = variable_get('ting_subsearch_kpi_account', NULL);
-    $password = variable_get('ting_subsearch_kpi_password', NULL);
-
-    if (empty($url) || empty($account) || empty($password)) {
+    if (empty($url)) {
       watchdog('ting_subsearch_opensuggestions', 'KPI: Module not configured properly', array(), WATCHDOG_WARNING);
       return;
     }
 
     try {
-      $client = new HttpClient();
-      $response = $client->get(
-        $url,
-        array(
-          'query' => $params,
-          'headers' => array(
-            'Accept' => 'application/json',
-          ),
-          'auth' => array(
-            $account,
-            $password,
-          ),
-        )
-      );
-      $result = json_decode($response->getBody(), TRUE);
+      // Fetch data from feed.
+      $ch = curl_init($url);
+      curl_setopt($ch, CURLOPT_RETURNTRANSFER, TRUE);
+      $ret = curl_exec($ch);
+      curl_close($ch);
+
+      $results = str_getcsv($ret, "\r\n");
+      unset($results[0]);
+
+      $prepared = array();
+
+      foreach ($results as $result) {
+        $result = explode(';', trim($result));
+        $prepared[] = array(
+          'weekno' => $result[0],
+          'oss' => $result[1],
+          'results' => $result[2],
+          'oss_r' => $result[3],
+        );
+      }
     }
-    catch (GuzzleClientException $e) {
-      throw new Exception($e->getMessage(), $e->getCode(), $e);
+    catch (Exception $e) {
+      watchdog('cron', $e->getMessage());
```

Comment:
Normally, the name of the 'type' parameter of the `watchdog` function is the module's name. Mixing with other modules can lead to confusion when revising reports.